
Let's wait untill we have gfxpayload support before we migrate to the new
loader.

Index: loader/i386/pc/linux.c
===================================================================
--- loader/i386/pc/linux.c	(revision 2233)
+++ loader/i386/pc/linux.c	(working copy)
@@ -382,10 +382,10 @@
 GRUB_MOD_INIT(linux16)
 {
   cmd_linux =
-    grub_register_command ("linux16", grub_cmd_linux,
+    grub_register_command ("linux", grub_cmd_linux,
 			   0, "load linux");
   cmd_initrd =
-    grub_register_command ("initrd16", grub_cmd_initrd,
+    grub_register_command ("initrd", grub_cmd_initrd,
 			   0, "load initrd");
   my_mod = mod;
 }
